migration-tools/utils.py

```diff
@@ -32,19 +32,25 @@ def migrate_hints(paragraph):
     for hint in hintRegex:
         hintSplit = hint.split("\n")
         hintType = re.search(r"'.*[']* %}", hintSplit[0]).group(0).replace("'", '').strip(" %}")
-        hintText = "\n".join(hintSplit[1:len(hintSplit)-1])
+        hintText = "\n".join(hintSplit[1:len(hintSplit)-1]).replace("    ", "")
         if hintType == 'note':
             hintType = 'tip'
 
-        newHint = f"{{{{% hints/{hintType} %}}}}\n{hintText}\n{{{{% /hints/{hintType} %}}}}"
+        newHint = f'\n\
+{{{{% hints/{hintType} %}}}}\n\
+  {hintText}\n\
+{{{{% /hints/{hintType} %}}}}'
         paragraph = paragraph.replace(hint, newHint)
 
     return paragraph
 
 def migrate_capture_alternative(paragraph):
     captureRE = re.findall(r"(?<={% capture alternative %})(.*?)(?= {% endcapture %})", paragraph, re.MULTILINE | re.DOTALL)
     for capture in captureRE:
-        info = f"{{{{% hints/info %}}}}\n{capture}\n{{{{% /hints/info %}}}}"
+        info = f'\n\
+{{{{% hints/info %}}}}\n\
+  {capture}\n\
```

Comment:
What are the two leading spaces for?
```suggestion
{capture}\n\
```

Reply:
Changed in [2d9c94a](https://github.com/arangodb/docs-hugo/pull/5/commits/2d9c94ab3f696535e35b9664b2de75dc76b56954)